classes/local/envbarlib.php

```diff
@@ -405,9 +405,10 @@ public function updatelastcheck() {
     /**
      * Sends a post to prod to update the lastrefresh time of this environment.
      *
+     * @param bool $force if true do not check prodlastping
      * @param bool $debug print curl debug if true
      */
-    public function pingprod($debug = false) {
+    public function pingprod($force = false, $debug = false) {
```

Comment:
This all looks good. A general comment is that I try to introduce new arguments at the end, not the start, to mitigate any risk of old usage of the api being changed. In this case there is only 2 places that call it so it's fine but it's a good habit